fabtests/scripts/runfabtests.sh

```diff
@@ -237,6 +238,10 @@ multinode_tests=(
 	"fi_multinode_coll"
 )
 
+prov_efa_tests=( \
+	"FI_EFA_RX_SIZE=1 fi_efa_ep_rnr_retry -R 0 -I 2"
```

Comment:
Is it possible to make `FI_EFA_RX_SIZE`  customized within the test? Seems other fabtests items do not have such envs defined explicitly in the scripts.